wiki/Building-From-Source.md

```diff
@@ -1,146 +1,103 @@
-ALVR can be built on Windows and Linux.
+ALVR can be built on Windows and Linux. The following instructions are for both OSes.
 
-# Windows
+# Common prerequisites
 
 Preferred IDE (optional): Visual Studio Code with rust-analyzer extension
 
-### Prerequisites
+You need to install [rustup](https://www.rust-lang.org/tools/install).
 
- * [Chocolatey](https://chocolatey.org/install)
- * [rustup](https://rustup.rs/). Alternatively install with: `choco install rustup.install` 
+On Windows you need also [Chocolatey](https://chocolatey.org/install).
 
-## Server
+# Server build
 
-**Note: These instructions are for the master branch.**
+First you need to gather some additional resources in preparation for the build.  
 
-On the repository root execute:
-```
-cargo xtask prepare-deps --platform windows
-cargo xtask build-server --release
-```
-ALVR server will be built into `build/alvr_server_windows/`.
+If you are on Linux, install these additional packages:
 
-To compile with software encoding support execute:
-```
-cargo xtask build-server --release --gpl
-```
-This will download and use FFmpeg binaries that are GPL licensed.
+* **Arch**
 
-## Client
+  ```bash
+  sudo pacman -Syu clang curl nasm pkgconf yasm vulkan-headers libva-mesa-driver unzip
```

Comment:
Arch's `ffmpeg` package from official repositories has all the things needed for alvr now, so that should probably be added here.
Also might want to mention the AUR package(s) as a automated way to build from source (AUR packages get built on your local system when you install them)

Reply:
Sorry but I struggle to navigate Linux stuff. Is it ok if we merge this PR and then you can do edits?

Reply:
Yeah